Makefile

```diff
@@ -21,6 +21,7 @@ TUNED_DIR:=daemon
 API_TYPES_DIR:=pkg/apis
 API_TYPES:=$(shell find $(API_TYPES_DIR) -name \*_types.go)
 API_ZZ_GENERATED:=zz_generated.deepcopy
+API_ZZ_GENERATED_FILES:=$(shell find $(API_TYPES_DIR) -name \$(API_ZZ_GENERATED)*)
```

Comment:
Could you provide an reproducer for me to try and see this is really needed?

Reply:
The reproducer will be:
1. Run `make pkg/generated`
2. Edit manually `pkg/apis/performanceprofile/v2/zz_generated.deepcopy.go`
3. Re-run `make pkg/generated`

Result:
It does not restore the `pkg/apis/performanceprofile/v2/zz_generated.deepcopy.go` to the correct format

In general, it is nothing that can not be fixed via `git` commands, but from generation tools, I am expecting that it will restore the content of generated files to have correct content, similar to files under `pkg/generated`.

Reply:
Would a simple change like
```
API_TYPES:=$(shell find $(API_TYPES_DIR) -name \*.go)
```
do what you want without adding extra complexity to the `Makefile`?  Perhaps adjusting the name of `API_TYPES` to something that more reflects what those sources are?